Index: out/admin/tpl/order_main.tpl
===================================================================
--- out/admin/tpl/order_main.tpl
+++ out/admin/tpl/order_main.tpl
@@ -186,17 +186,25 @@
                         <tr>
                             <td class="edittext">[{ oxmultilang ident="ORDER_MAIN_DELTYPE" }]:</td>
                             <td class="edittext">
+                                [{ if $edit->oxorder__oxpaymenttype->value != "rakuten" }]
                                 <select name="setDelSet" class="editinput" style="width: 135px;">
                                 <option value="">----</option>
                                 [{foreach from=$oShipSet key=sShipSetId item=oShipSet}]
                                 <option value="[{ $sShipSetId }]" [{ if $edit->oxorder__oxdeltype->value == $sShipSetId }]SELECTED[{/if}]>[{$oShipSet->oxdeliveryset__oxtitle->value}]</option>
                                 [{/foreach}]
                                 </select>
+                                [{else}]
+                                    [{foreach from=$oShipSet key=sShipSetId item=oShipSet}]
+                                        [{ if $edit->oxorder__oxdeltype->value == $sShipSetId }][{$oShipSet->oxdeliveryset__oxtitle->value}][{/if}]
+                                    [{/foreach}]
+                                [{/if}]
                                 [{ oxinputhelp ident="HELP_ORDER_MAIN_DELTYPE" }]
                             </td>
                             <td>
+                                [{ if $edit->oxorder__oxpaymenttype->value != "rakuten" }]
                                 <input type="submit" class="edittext" name="save" id="shippNowButton" onclick="document.sendorder.sendmail.value=document.getElementById('sendmail').checked;document.sendorder.submit();return false;" value="&nbsp;&nbsp;[{ oxmultilang ident="GENERAL_NOWSEND" }]&nbsp;&nbsp;" [{ $readonly }]>
                                 <input id='sendmail' class="edittext" type="checkbox" name="sendmail" value='1' [{ $readonly }]> [{ oxmultilang ident="GENERAL_SENDEMAIL" }]
+                                [{/if}]
                             </td>
                         </tr>
                         <tr>
@@ -207,7 +215,9 @@
                             <b>[{ $edit->oxorder__oxsenddate->value|oxformdate:'datetime':true }]</b>
                             </td>
                             <td>
+                            [{ if $edit->oxorder__oxpaymenttype->value != "rakuten" }]
                             <input type="submit" class="edittext" name="save" id="resetShippingDateButton" value="[{ oxmultilang ident="GENERAL_SETBACKSENDTIME" }]" onclick="document.resetorder.submit();return false;" [{ $readonly }]>
+                            [{/if}]
                             </td>
                         </tr>
                     </table>
Index: out/admin/tpl/order_overview.tpl
===================================================================
--- out/admin/tpl/order_overview.tpl
+++ out/admin/tpl/order_overview.tpl
@@ -261,6 +261,7 @@
         <br>
         [{ if $edit }]
         <table cellspacing="0" cellpadding="0" border="0">
+        [{if !$oView->canResetShippingDate() }]
         <form name="sendorder" id="sendorder" action="[{ $oViewConf->getSelfLink() }]" method="post">
         [{ $oViewConf->getHiddenSid() }]
         <input type="hidden" name="cl" value="order_overview">
@@ -272,11 +273,23 @@
                 <td class="edittext">
                 </td>
                 <td class="edittext" style="border : 1px #A9A9A9; border-style : solid solid solid solid; padding-top: 5px; padding-bottom: 5px; padding-right: 5px; padding-left: 5px;">
+                    [{ if $edit->oxorder__oxpaymenttype->value == "rakuten" }]
+                    [{ oxmultilang ident="ORDER_MAIN_DELTYPE" }]:
+                        <select name="setDelSet" class="editinput" style="width: 135px;">
+                            [{foreach from=$oShipSet key=sShipSetId item=oShipSet}]
+                            <option value="[{$oShipSet}]" >[{$oShipSet}]</option>
+                            [{/foreach}]
+                        </select>
+                        <br/>
+                        [{ oxmultilang ident="ORDER_MAIN_TRACKCODE" }]: <input type="text" class="editinput" size="25" maxlength="[{$edit->oxorder__oxtrackcode->fldmax_length}]" name="oxorder__oxtrackcode" value="[{$edit->oxorder__oxtrackcode->value }]">
+                        <br/>
+                    [{/if}]
                     <input type="submit" class="edittext" name="save" value="&nbsp;&nbsp;[{ oxmultilang ident="GENERAL_NOWSEND" }]&nbsp;&nbsp;" [{ $readonly }]><br>
                     [{ oxmultilang ident="GENERAL_SENDEMAIL" }] <input class="edittext" type="checkbox" name="sendmail" value='1' [{ $readonly }]>
                 </td>
             </tr>
             </form>
+            [{/if}]
             <tr>
                 <td class="edittext">
                 </td>
@@ -289,7 +302,7 @@
                 </td>
             </tr>
         [{/block}]
-        [{ if $oView->canResetShippingDate() }]
+        [{ if $oView->canResetShippingDate() AND $edit->oxorder__oxpaymenttype->value != "rakuten" }]
         <form name="resetorder" id="resetorder" action="[{ $oViewConf->getSelfLink() }]" method="post">
         [{ $oViewConf->getHiddenSid() }]
         <input type="hidden" name="cl" value="order_overview">
Index: out/basic/tpl/page/checkout/basket.tpl
===================================================================
--- out/basic/tpl/page/checkout/basket.tpl
+++ out/basic/tpl/page/checkout/basket.tpl
@@ -23,6 +23,7 @@
     [{if $oView->isLowOrderPrice() }]
       <div class="minorderprice">[{ oxmultilang ident="BASKET_MINORDERPRICE" }] [{ $oView->getMinOrderPrice() }] [{ $currency->sign }]</div>
     [{else}]
+    [{block name="basket_btn_next_top"}]
     <form action="[{ $oViewConf->getSslSelfLink() }]" method="post">
       <div>
           [{ $oViewConf->getHiddenSid() }]
@@ -32,6 +33,7 @@
           </div>
       </div>
     </form>
+    [{/block}]
     [{/if}]
   </div>

@@ -463,7 +465,9 @@
     [{if $oView->isLowOrderPrice() }]
       <div class="minorderprice">[{ oxmultilang ident="BASKET_MINORDERPRICE" }] [{ $oView->getMinOrderPrice() }] [{ $currency->sign }]</div>
     [{else}]
+
     <form action="[{ $oViewConf->getSslSelfLink() }]" method="post">
+      [{block name="basket_btn_next_bottom"}]
       <div>
           [{ $oViewConf->getHiddenSid() }]
           <input type="hidden" name="cl" value="user">
@@ -472,6 +476,7 @@
           </div>
       </div>
     </form>
+    [{/block}]
     [{/if}]

   </div>
